### DISCUSSION TITLE
运行bfe，提示“maxprocs: Leaving GOMAXPROCS=1: CPU quota undefined”错误

### DISCUSSION THREAD

**Discussion by lijl-vb** [bfenetworks/bfe#1124] [Q&A (问题和回答)]
**Describe the bug**
运行bfe，提示“maxprocs: Leaving GOMAXPROCS=1: CPU quota undefined”错误。

bfe版本：bfe_1.6.0_linux_amd64

**Screenshots**

![image](https://github.com/bfenetworks/bfe/assets/73126845/274576db-a00d-46b6-96d0-1259b870ca8e)

**Comment by youerning** [comment]
这只是一个警告，不用慌。
这个警告来自'go.uber.org/automaxprocs'这个库，它是为了避免golang的程序在docker环境下CPU被限制的情况下造成golang性能下降，这点跟golang的GMP模型有关，你可以搜索一下这个库的使用说明就了解了。
